azext_iot/digitaltwins/providers/resource.py

```diff
@@ -230,9 +264,153 @@ def remove_role(self, name, assignee, role_type=None, resource_group_name=None):
             dt_scope=target_instance.id, assignee=assignee, role_type=role_type
         )
 
+    # Identity
+
+    def assign_identity(
+        self,
+        name: str,
+        system_identity: Optional[bool] = None,
+        user_identities: Optional[List[str]] = None,
+        identity_role: Optional[str] = None,
+        identity_scopes: Optional[List[str]] = None,
+        resource_group_name: Optional[str] = None
+    ):
+        target_instance = self.find_instance(
+            name=name, resource_group_name=resource_group_name
+        )
+        if not resource_group_name:
```

Comment:
add test for missing rg for assign_identity and remove_identity. Can be unit test